aten/src/ATen/native/quantized/QTensor.cpp

```diff
@@ -72,5 +72,13 @@ Tensor per_tensor_affine_qtensor_cpu(const Tensor& self, double scale, int64_t z
   return dst;
 }
 
+Tensor& qtensor_set_storage_cpu(Tensor& self, Storage storage, int64_t storage_offset, IntArrayRef sizes, IntArrayRef strides) {
```

Comment:
also, doesn't it work through generic impl? QTensorImpl inherits from regular TensorImpl so the following 3 methods should work in theory

Reply:
it works using unsafeGetTensorImpl() as well. will change to that.